plugins/search/src/apis.ts

```diff
@@ -35,12 +37,18 @@ class SearchApi {
 
   private async entities() {
     const entities = await this.catalogApi.getEntities();
-    return entities.items.map((result: any) => ({
-      name: result.metadata.name,
-      description: result.metadata.description,
-      owner: result.spec.owner,
-      kind: result.kind,
-      lifecycle: result.spec.lifecycle,
+    return entities.items.map((entity: Entity) => ({
+      name: entity.metadata.name,
+      description: entity.metadata.description || 'No description',
+      owner: typeof entity.spec?.owner ?? undefined,
```

Comment:
You accidentally deleted a bit much here

Reply:
ups, and I also did not take away the fallback for description which I planned to

Reply:
fixed!